buttercup/cogs/history.py

```diff
@@ -92,6 +92,18 @@ def add_rank_lines(ax: plt.Axes, gamma: int) -> plt.Axes:
     return ax
 
 
+def add_rate_lines(ax: plt.Axes, max_rate: int) -> plt.Axes:
+    """Add the rate lines to the given axes."""
+    # Show rate milestones when you are close to them already
+    threshold_buffer = 40
+    for index, rank in enumerate(ranks[3:]):
+        # A milestone every 100 units
+        rate_milestone = (index + 1) * 100
+        if rate_milestone - max_rate - threshold_buffer <= 0:

```

Comment:
I feel like `max_rate + threshold_buffer >= rate_milestone` is a more clear expression

Reply:
This got indirectly fixed by the refactor.